File: Patterns/printPattern8.java
import java.util.*;

public class Main {
    public static void printPattern8(int row) {
        for(int i = 1; i <= row; i++) {
            for(int j=1; j<=row; j++){//for(int j = row; j>=1; j--) {
                if(i == j) {
                    System.out.print("*\t");
                } else {
                    System.out.print("\t");
                }
            }
            System.out.println();
        }
    }

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int row = scn.nextInt();
        printPattern8(row);
        // write ur code here

    }
}

File: Patterns/printPattern8.cpp
#include <iostream>
using namespace std;
void pattern8(int row){
    int nst=1;
    int nsp=row-1;
    for(int r=1; r<=row; r++){
        for(int csp=1; csp<=nsp; csp++){
            cout<<"\t";
        }
        for(int cst=1; cst<=nst; cst++){
            cout<<"*\t";
        }
        
        nsp--;
        cout<<endl;
    }
}
int main(int argc, char **argv){
    int n;
    cin >> n;
    pattern8(n);
    //write your code here
    
}